src/dists.ad.js

```diff
@@ -228,7 +229,12 @@ var Uniform = makeDistributionType({
 
 var UniformDrift = makeDistributionType({
   name: 'UniformDrift',
-  params: [{name: 'a'}, {name: 'b'}, {name: 'r', desc: 'drift kernel radius'}],
+  desc: 'Drift version of Uniform. ' +
+      'Drift kernels are used to narrow search during inference. ' +
+      'UniformDrift proposes from a symmetric window around the current value x, [x-r, x+r]',
+  params: [{name: 'a', desc: 'Lower bound (real)'},
+	   {name: 'b', desc: 'Upper bound (real > a)'},
```

Comment:
It would be good to standardize the capitalization of parameter descriptions, and either always capitalize or never. The current docs do the latter.